charts/lighthouse/templates/webhooks-ingress.yaml

```diff
@@ -8,13 +17,25 @@ metadata:
   annotations:
     {{- toYaml .Values.webhooks.ingress.annotations | nindent 4 }}
 spec:
+  {{- if .Values.webhooks.ingress.ingressClassName }}
+  ingressClassName: {{ .Values.webhooks.ingress.ingressClassName }}
+  {{- end }}
   rules:
   {{- range .Values.webhooks.ingress.hosts }}
   - host: {{ . | quote }}
     http:
       paths:
         - backend:
```

Comment:
Can we keep the logic at a single place ?

Reply:
IngressClassName was added in 1.18 and was not part of the stable vs beta so it doesn't pin to the apiVersion directly.

Reply:
Maybe just add `{{- if semverCompare ">=1.18-0" .Capabilities.KubeVersion.Version }}` ?

Reply:
You would still need to set a value though, otherwise it would be blank

Reply:
Just set `ingressClassName: null` in `values.yaml`

Reply:
updated as well